packages/video_player/video_player_avfoundation/example/lib/main.dart

```diff
@@ -119,8 +119,7 @@ class _BumbleBeeRemoteVideoState extends State<_BumbleBeeRemoteVideo> {
   void initState() {
     super.initState();
     _controller = MiniController.network(
-      'https://flutter.github.io/assets-for-api-docs/assets/videos/bee.mp4',
-    );
+        'https://flutter.github.io/assets-for-api-docs/assets/videos/bee.mp4');
```

Comment:
revert

Reply:
Reverted, forgot the comma, therefore it was modified